### PR TITLE
DEVELOPER-1165 Added in a details section if available and added the sec...

### PR DESCRIPTION
**Pull request by Dantheman720**
...tion for data virtualization

### CODE REVIEW

File: _layouts/product-overview.html.slim [hunk 1]
```diff
@@ -27,10 +27,15 @@ br
   div(class="feature-text")
     = page._features3
 
+= page._new_feature
```
Comment (paulrobinson) [inline _layouts/product-overview.html.slim]: You don't need to do this. Just add the content to `overview.adoc` as a new level-3 (===) section at the end of the `detail` section.

File: products/datavirt/overview.adoc [hunk 2]
```diff
@@ -49,4 +49,29 @@ JBoss Data Virtualization offers comprehensive data abstraction, federation, int
...
-== Resources 
+==== For JBoss Data Virtualization 6.1 Beta, we focused on
+
+- Big Data
+- Cloud

+- Development and Deployment Improvements

+
+==== BIG DATA
```
Comment (paulrobinson) [inline products/datavirt/overview.adoc]: This shouldn't be caps (Big Data)

File: products/datavirt/overview.adoc [hunk 3]
```diff
@@ -49,4 +49,29 @@ JBoss Data Virtualization offers comprehensive data abstraction, federation, int
...
+
+- Support for Cloudera Impala in addition to Apache Hive
+- Support for Apache Solr as a data source
+- Full support for MongoDB 
+- Expanded support for JDG with the ability to perform writes Apache
+- Cassandra (Tech Preview)
+
+==== CLOUD
```
Comment (paulrobinson) [inline products/datavirt/overview.adoc]: more caps

File: products/datavirt/overview.adoc [hunk 4]
```diff
@@ -49,4 +49,29 @@ JBoss Data Virtualization offers comprehensive data abstraction, federation, int
...
+- OpenShift Online with new WebUI (Dev Preview)
+- Cloud enablement with support for Amazon EC2 and Google Compute Engine
+
+==== PRODUCTIVITY AND DEPLOYMENT IMPROVEMENTS
```
Comment (paulrobinson) [inline products/datavirt/overview.adoc]: more caps

**Comment by paulrobinson** [comment]
retest this please

File: _layouts/product-overview.html.slim [hunk 5]
```diff
@@ -31,6 +31,9 @@ br
 
 = page._detail
 
+- unless !page._detail || page._detail.empty?
```
Comment (paulrobinson) [inline _layouts/product-overview.html.slim]: I think you need to remove the changes to this file, right?

**Comment by Dantheman720** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Pull request merged**